CHANGELOG.rst

```diff
@@ -29,7 +29,7 @@ Deprecated
 Fixed
 -----
 - `[Core Framework] clang-tidy improvement for CMakeResourceDependencies <https://github.com/CppMicroServices/CppMicroServices/pull/812>`_
-- `[Core Framework] GetService performance micro-optimizations <https://github.com/CppMicroServices/CppMicroServices/pull/833>`_
+- `[Declarative Services] fixed sporadic crash caused by concurrrent modification of the ComponentMgrImpl vector <https://github.com/CppMicroServices/CppMicroServices/pull/834>`_
```

Comment:
`[Core Framework] GetService performance micro-optimizations <https://github.com/CppMicroServices/CppMicroServices/pull/833>`_ was removed and needs to be added back